android/src/main/java/slayer/accessibility/service/flutter_accessibility_service/AccessibilityListener.java

```diff
@@ -136,8 +138,10 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
     @RequiresApi(api = Build.VERSION_CODES.N)
     @TargetApi(Build.VERSION_CODES.LOLLIPOP)
-    void getSubNodes(AccessibilityNodeInfo node, List<HashMap<String, Object>> arr) {
+    void getSubNodes(AccessibilityNodeInfo node, List<HashMap<String, Object>> arr, List<AccessibilityNodeInfo> traversedNodes) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            if (traversedNodes.contains(node)) return;
```

Comment:
The override gives me confidence that this should work well.
<img width="665" alt="image" src="https://github.com/X-SLAYER/flutter_accessibility_service/assets/41507081/24c104be-d87b-4481-93fb-7d1d2fb3faef">

Reply:
Since I don't have a way of replicating the issue on demand